context-core/src/main/java/de/atb/context/tools/ontology/utils/ConfigurationsUtilities.java

```diff
@@ -30,20 +29,12 @@
     private static final Logger LOGGER = LoggerFactory
             .getLogger(ConfigurationsUtilities.class);
     static String deployableConfigStringId = "Task_";
-    static KMBApi kmbapi = new KMBApi();
     static String configNamespace = ("ContextHandling_Ontological_Model:");
 
     private ConfigurationsUtilities() {}
```

Comment:
The whole class `ConfigurationsUtilities` can be removed, too. It is not used anywhere.

Same is true for `KMBConfigsVocabulary`.

Reply:
done with the latest commit